File: src/tests/blend_test/blend_test.py
import pyccante as pyc

blend = pyc.Image("blend.png", pyc.LDR_type.LT_NONE)
to_blend = pyc.Image("to_blend.png", pyc.LDR_type.LT_NONE)
weight = pyc.Image("weight.png", pyc.LDR_type.LT_NONE)

blend.blend(to_blend, weight)
blend.Write("blended.png", pyc.LDR_type.LT_NONE)

File: examples/filter_bilateral_2df_test/blend_test/blend_test.cpp
// compile with: g++ -std=c++17 -I/usr/include/GL -Iusr/include/GLFW -Iusr/include/GLEW -I../../include/stb -I../../../include/piccante/include blend_test.cpp -lGL -lGLEW -lGLU -lglut -lz

#include <GL/glew.h>
#include <piccante.hpp>

int main(){

    pic::Image blend("blend.png", pic::LDR_type::LT_NONE);
    pic::Image to_blend("to_blend.png", pic::LDR_type::LT_NONE);
    pic::Image weight("weight.png", pic::LDR_type::LT_NONE);

    blend.blend(&to_blend, &weight);
    blend.Write("blended.png", pic::LDR_type::LT_NONE);

}